sockeye/scoring.py

```diff
@@ -0,0 +1,481 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License
+# is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+Implement scoring of existing translations.
```

Comment:
I think here or somewhere in the help string for this CLI it should be made a bit clearer what scoring means for the user, i.e. what are the scores returned and how are they obtained.

Reply:
My newest commit contains more information here. Let me know if this is what you expected.